src/aiortc/contrib/media.py

```diff
@@ -223,6 +224,10 @@ def __init__(self, file, format=None, options={}):
         self.__thread: Optional[threading.Thread] = None
         self.__thread_quit: Optional[threading.Event] = None
 
+        # check whether we need to throttle playback
+        container_format = set(self.__container.format.name.split(","))
+        self._throttle_playback = not container_format.intersection(REAL_TIME_FORMATS)
```

Comment:
Are you sure we need this change? There is no way the `recv` methods of the tracks can be called before the constructor completes, so I don't see what difference it makes setting this earlier